### DISCUSSION TITLE
/v1/workflow-instances/{workflowInstanceId}/execute problem

### DISCUSSION THREAD

**Discussion by tomy2105** [elsa-workflows/elsa-core#3095] [Design & tech]
Can /v1/workflow-instances/{workflowInstanceId}/execute be used to continue execution of an instance which is suspended in http endpoint activity? 
If yes, is there a way to use it without specifying activity id (I ask because without specifying activity id my request fails due to null exceptions)? 
If no could you please explain for what should /v1/workflow-instances/{workflowInstanceId}/execute be used? 
What activity must be used to Suspend workflow instance in order to make it run again using /v1/workflow-instances/{workflowInstanceId}/execute?

**Comment by sfmskywalker** [comment]
Yes, and technically, yes should be possible if we make a small change - however, without specifying the activity ID, you run the risk of the engine picking the wrong activity ID in case the workflow instance is blocked on multiple activities.
